<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ftp_uploadClass</class>
 <widget class="QMainWindow" name="ftp_uploadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ftp_upload</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>561</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>541</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string>上传文件：</string>
     </property>
     <widget class="QTextEdit" name="textEditTest">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>521</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpload">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>开始上传</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonChoose">
         <property name="text">
          <string>选择json文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPath"/>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>54</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>上传结果：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>541</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>ftp配置信息</string>
     </property>
     <widget class="QLabel" name="labelFtpConfig">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>19</y>
        <width>511</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>560</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionShowLog"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfig">
   <property name="text">
    <string>配置ftp</string>
   </property>
  </action>
  <action name="actionShowLog">
   <property name="text">
    <string>查看日志</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ftp_upload.qrc"/>
 </resources>
 <connections/>
</ui>
